File: Classes/CoreData/Objects/CDFile.m
//
//  CDFile.m
//  Dokie
//
//  Created by Ashamaz Shidov on 18/01/2020.
//  Copyright © 2020 Ashamaz Shidov. All rights reserved.
//

#import "CDFile.h"

@implementation CDFile

@dynamic identifier;
@dynamic data;

#pragma mark -
static NSString *_kIdentifier = @"identifier";
+ (NSString *)kIdentifier { return _kIdentifier; }

static NSString *_kData = @"data";
+ (NSString *)kData { return _kData; }

@end

File: Classes/CoreData/Objects/CDFile.swift
//
//  CDFile.swift
//  Dokie
//
//  Created by Ashamaz Shidov on 16.05.2020.
//  Copyright © 2020 Ashamaz Shidov. All rights reserved.
//

import CoreData

class CDFile : NSManagedObject {
    
    @NSManaged var data: Data
    @NSManaged var identifier: String

    static let kData = "data"
    static let kIdentifier = "identifier"

}
